calico_node/filesystem/etc/rc.local

```diff
@@ -24,8 +26,10 @@ else
     # running twice ensures our starting configuration is correct.
```

Comment:
Now that we're printing logs here, so we want to still call it "policy-only mode"?  Just a thought per the discussions we had about renaming that.